.github/workflows/shared_integration_tests.yaml

```diff
@@ -240,29 +232,40 @@ jobs:
               AWS_SECRET_ACCESS_KEY: ${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}
               AWS_DEFAULT_REGION: 'us-east-1'
 
-          - name: Publish allure report link to the PR
-            id: gh_comment_allure_link
-            if: ${{ env.SKIP_TESTS != 'true' && steps.upload_allure_report.outcome == 'success' }}
-            continue-on-error: true  # if this fails, we still need to run clean-up steps
-            run: gh pr comment $PR_NUM --body "Please find the detailed integration test report [here](https://qa.planx-pla.net/dashboard/Secure/gen3-ci-reports/$REPO/$PR_NUM/$RUN_NUM/index.html)" -R $REPO_FN
-            env:
-              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
           - name: Archive pod logs from CI environment
             id: archive_pod_logs
             if: ${{ env.SKIP_TESTS != 'true' && steps.prep_ci_env.outcome == 'success' || steps.prep_ci_env.outcome == 'failure' }}
             continue-on-error: true  # if this fails, we still need to run clean-up steps
             run: poetry run python -m gen3_ci.scripts.save_ci_env_pod_logs
 
-          - name: Publish pod logs url to the PR
-            id: gh_comment_pod_logs_link
-            if: ${{ env.SKIP_TESTS != 'true' && steps.archive_pod_logs.outcome == 'success' }}
+          - name: Generate markdown report
+            id: generate_md_report
+            if: ${{ env.SKIP_TESTS != 'true' && steps.generate_allure_report.outcome == 'success' || steps.archive_pod_logs.outcome == 'success' }}
+            continue-on-error: true
+            # It is possible for env is prepped but tests error out, the pod logs will help in debugging env issues
+            run: |
+              if [ "${{ steps.upload_allure_report.outcome }}" == "success" ]; then
+                echo -e "\nPlease find the detailed integration test report [here](https://qa.planx-pla.net/dashboard/Secure/gen3-ci-reports/$REPO/$PR_NUM/$RUN_NUM/index.html)\n" >> output/report.md
+              fi
+              if [ "${{steps.archive_pod_logs.outcome}}" == "success" ]; then
+                if [ ! -d output ]; then
+                  mkdir output
+                fi
+                if [ ! -f "output/report.md" ]; then
+                  touch "output/report.md"
+                fi
```

Comment:
should these commands (creating of `output` dir) be moved to before adding the allure report to `output/report.md` on line 248?

Reply:
It is possible that the env is up but
i) test run never started
ii) tests ran but never completed (errored out)
It would be useful to report the link to the pod logs in these scenarios too